src/pages/profile/ui/component.tsx

```diff
@@ -1,9 +1,8 @@
 import { useStore } from 'effector-react';
-import { sessionModel } from 'entities/session';
 import React from 'react';
 import { Link } from 'react-router-dom';
 import { Box, Button, Container, Stack, Typography } from 'shared/ui';
-import { $user } from '../model';
+import { $user, loginRequested, logoutRequested } from '../model';
```

Comment:
we export and import models via public API, in this case, the model from store.ts better import * as model './model';